torch/_inductor/codegen/common.py

```diff
@@ -511,6 +511,139 @@ def remainder(a, b):
     def load_seed(name, offset):
         return ops.load(name, sympy.Integer(offset))
 
+    @classmethod
+    def _initialize_pointwise_overrides(cls, target):
+        assert target in {"triton", "cpp"}, target
+
+        def pointwise_factory(impl):
+            def func(x):
+                return f"{impl}({x})"
```

Comment:
This won't work for `CppVecOverrides` since most operations are methods on the `Vectorized` type. It's also called pointwise but is really only unary pointwise.

Reply:
Fixed via introducing `cppvec` item to `pointwise_overrides_data`.